modules/serverless-rn-1-29-1.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-29-1_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.29.1
+
+{ServerlessProductName} 1.29.1 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-29-1_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.8.
+* {ServerlessProductName} now uses Knative Eventing 1.8.
+* {ServerlessProductName} now uses Kourier 1.8.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.8.1.
+* {ServerlessProductName} now uses Knative Kafka 1.8.0.
+* The `kn func` CLI plug-in now uses `func` 1.10. FIXME is this correct?
+
+* This release of {ServerlessProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on OpenShift Container Platform 4.10 and later versions.
```

Comment:
Is it worth moving this bullet outside "New feature"? It sounds like a general description of this release but it is under "New features" as one of the features. I'd probably put it right below https://github.com/openshift/openshift-docs/pull/62581/files#diff-8c685f7d3fdd7f35beb4bf6f637e91fe83d0276201f288549403ec84230bd849R9